Dockerfile

```diff
@@ -21,6 +23,9 @@ RUN Rscript -e "pak::local_install_deps('/workflow.pacta.dashboard')"
 
 FROM base AS install-pacta
 
+ENV DASHBOARD_FILES_DIR="/mnt/dashboard_files"
```

Comment:
I would name this something more descriptive... so far we have:
  dashboard_data_dir = Sys.getenv("DASHBOARD_DATA_DIR"),
  dashboard_files_dir = Sys.getenv("DASHBOARD_FILES_DIR"),
  dashboard_output_dir = Sys.getenv("DASHBOARD_OUTPUT_DIR")

Which is a bit confusing. Maybe `dashboard_build_files` would be more clear?